rudder-web/src/main/webapp/index.html

```diff
@@ -2,60 +2,132 @@
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
+<meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
 <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
+    
 <title>Rudder - Login</title>
-<link href="/style/rudder-login.css" rel="stylesheet" type="text/css" media="screen" />
-<link href="/images/themes/ui-lightness/jquery.ui.all.css" rel="stylesheet" type="text/css" />
 
+<link href="/images/themes/ui-lightness/jquery.ui.all.css" rel="stylesheet" type="text/css" />
+<link rel="stylesheet" href="/bootstrap-3.2.0-dist/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
```

Comment:
Hi. Do you really want to include code on the login page from the internet? What about environments, where there is no internet available for the Browser you are using the Rudder UI?... might happen...